src/dirs.proj

```diff
@@ -12,6 +12,22 @@
     <Warning Text="Looks like there may be an update to the build tools. Please run build.cmd from the root of the repo to refresh the build tools." /> 
   </Target>
 
+  <ItemGroup>
+    <!-- Include test runtime when restoring packages in EnsureDependencies -->
+    <PackageConfigs Include="$(BaseIntermediateOutputPath)test-runtime\packages.config" />
+  </ItemGroup>
+  
+  <Target Name="PrepareTestRuntimeRestore" BeforeTargets="EnsureDependencies">
+    <Copy
+      SourceFiles="$(ToolsDir)test-runtime-packages.config"
+      DestinationFiles="$(BaseIntermediateOutputPath)test-runtime\packages.config"
+      SkipUnchangedFiles="$(SkipCopyUnchangedFiles)"
+      OverwriteReadOnlyFiles="$(OverwriteReadOnlyFiles)"
+      Retries="$(CopyRetryCount)"
+      RetryDelayMilliseconds="$(CopyRetryDelayMilliseconds)"
+      UseHardlinksIfPossible="$(CreateHardLinksForCopyFilesToOutputDirectoryIfPossible)" />
+  </Target>
```

Comment:
I think making changes to nuspec and or the test-runtime/package.config file makes the most since. We should try to avoid copying or renaming files as a post step to installing the build tools package. We should try to get the layout of the package to exactly match what we can consume so we can consume it directly from their unmodified.